src/test/javascript/CommonJsTest.js

```diff
@@ -78,6 +78,71 @@ QUnit.test('sortDate(x, y)', function(assert) {
         '25 April 1999 2:00PM - 25 April 1999 2:01PM');
 });
 
+QUnit.test('getPointValue(s, ditchZero)', function(assert) {
```

Comment:
Table sorting is used in almost 100 classes. I'm hesitant too. There doesn't seem to be any harm keeping it.